CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.6.3
+-fix visibility of data overlays on parent objects
+-add entity binding support
```

Comment:
I don't know what these changes are referring to

Reply:
Weston is not here today, However these are AppKit changes.

Reply:
This refer to the entity binding support introduced in 3DKG previously binding was supported only through widgets like tags ..etc no you can do entity binding without the needs for widgets like tags ..etc.